### PR TITLE
Always dispatch `amp:load:end`

### LINKED ISSUE #5862: amp-sticky-ad fails to load sometimes on Safari 10
Fails about 1 in 3 times. 
Sample URL: https://fresnobee.relaymedia.com/amp/news/local/education/article61889207.html
Note that browser emulation works fine - only when tried from an iPhone using Safari 10.

CC @zhouyx

### PR DESCRIPTION
**Pull request by zhouyx**
fix #5862

### CODE REVIEW

File: src/custom-element.js [hunk 1]
```diff
@@ -1041,8 +1041,8 @@ function createBaseCustomElementClass(win) {
         if (this.isFirstLayoutCompleted_) {
           this.implementation_.firstLayoutCompleted();
           this.isFirstLayoutCompleted_ = false;
-          this.dispatchCustomEvent('amp:load:end');
         }
+        this.dispatchCustomEvent('amp:load:end');
```
Comment (dvoytenko) [inline src/custom-element.js]: Please explain why?

**Comment by zhouyx** [comment]
>>>
it seems #getResourcesInViewport run before sticky-ad build. During that time we haven't declare ownership for the ad yet, and the ad will be laid out. However sticky-ad assumes its child ad can only be laid out with it manually call #scheduleLayout. It never receive load:end event in this case.

**Comment by dvoytenko** [comment]
This explanation doesn't yet make sense to me. What does `getResourcesInViewport` have to do with this?

**Comment by zhouyx** [comment]
Close this because it doesn't solve the real issue to the bug

**Pull request closed without merging**